data_steward/ci/upload_cloud_creds.sh

```diff
@@ -46,4 +46,4 @@ echo "----- Created vars:"
 curl https://circleci.com/api/v1.1/project/$VCS_TYPE/$CI_USERNAME/$CI_PROJECT/envvar?circle-token=$CI_TOKEN
 
 echo "----- Decryption command:"
-echo 'echo $GCLOUD_CREDENTIALS | openssl enc -d -aes-256-cbc -base64 -A -k $GCLOUD_CREDENTIALS_KEY'
+echo "echo \$GCLOUD_CREDENTIALS | openssl enc -d -aes-256-cbc -base64 -A -md md5 -k \$GCLOUD_CREDENTIALS_KEY"
```

Comment:
Should the openssl in line 49 match line 32? They are slightly different.

Reply:
@amcgrenera-vumc I don't think so.  I believe Line 32 is creating the encrypted SSL key.  Line 49 is echoing the command to decrypt the key.  So they should be different.